Cargo.toml

```diff
@@ -10,7 +10,7 @@ resolver = "2"
 [workspace.package]
 version = "0.1.0"
```

Comment:
Will you shape the release as a separate PR?
Anyways, you should have the rights to publish the package once released.

Reply:
Yes, I'll do it separately.